src/server/helpers/middleware.js

```diff
@@ -23,6 +23,8 @@ import * as commonUtils from '../../common/helpers/utils';
 import * as error from '../../common/helpers/error';
 import * as utils from '../helpers/utils';
 import type {$Request, $Response, NextFunction} from 'express';
+import Promise from 'bluebird';
```

Comment:
Same comment as on package.json, bluebird has been removed and this line shouldn't be needed.